.yamato/config/vfx_hdrp.metafile

```diff
@@ -31,6 +31,16 @@ test_platforms:
     utr_flags:
       - [all]: --testfilter="VFX.Test"
       - [all]: --extra-editor-arg="-xr-reuse-tests"
+  - type: playmode
+    name: playmode_XR_cache
+    utr_flags:
+    - [all]: --extra-editor-arg="-xr-reuse-tests"
+    - [all]: --extra-editor-arg="-adb2"
+    - [all]: --extra-editor-arg="-enableCacheServer"
+    - [all]: --extra-editor-arg="-cacheServerEndpoint" --extra-editor-arg="cacheserver-slo.hq.unity3d.com"
+    - [all]: --extra-editor-arg="-cacheServerNamespacePrefix" --extra-editor-arg="SRP"
+    - [all]: --extra-editor-arg="-cacheServerEnableDownload" --extra-editor-arg="true"
+    - [all]: --extra-editor-arg="-cacheServerEnableUpload" --extra-editor-arg="true"
```

Comment:
Here, `--testfilter="VFX.Test"` is missing (to skip performanceTest)
It will conflict with this https://github.com/Unity-Technologies/Graphics/pull/203

Reply:
fixed

Reply:
Confirmed it has been fixed at https://github.com/Unity-Technologies/Graphics/pull/3215/commits/824d8405de56801ec47e36d1d4818d990b8099e0, thanks for the one in vfx_urp, I missed it.